app/src/main/java/com/nervousfish/nervousfish/activities/ContactReceivedHelper.java

```diff
@@ -41,17 +45,22 @@
      * @param contact  The newly received {@link Contact}
      */
     static void newContactReceived(final IDatabase database, final Activity activity, final Contact contact) {
+        if (contact == null) {
```

Comment:
Validate.notNull(database);
Validate.notNull(activity);
Validate.notNull(contact);
Same amount of lines, but 3 times more checks